include/ubiquity_motor/motor_hardware.h

```diff
@@ -50,6 +50,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <gtest/gtest_prod.h>
 
+// Mininum hardware versions required for various features
+#define MIN_HW_OPTION_SWITCH 50
```

Comment:
For historical reasons and visibility all of these sort of constants are all starting around line 45 of  motor_msg.h  so this would be a break in 'the standard' to have this one here.

Reply:
That's the same thought process I went through :)